airbyte-cdk/python/airbyte_cdk/sources/concurrent_source/concurrent_source_adapter.py

```diff
@@ -58,3 +68,60 @@ def _select_abstract_streams(self, config: Mapping[str, Any], configured_catalog
             if isinstance(stream_instance, AbstractStreamFacade):
                 abstract_streams.append(stream_instance.get_underlying_stream())
         return abstract_streams
+
+    def _convert_to_concurrent_stream(self, logger: logging.Logger, stream: Stream, cursor: Optional[Cursor] = None) -> Stream:
```

Comment:
It feels like we should make this a public method. It's invoked or overwritten in both of the concurrent sources so rather than make it invoked privately, it feels more transparent to expose it

Reply:
since we invoke it for each concurrent source, it seems reasonable to change it to public.